src/open_inwoner/accounts/query.py

```diff
@@ -1,9 +1,9 @@
-from django.db.models import Case, F, Max, OuterRef, Q, Subquery, Value, When
+from django.db.models import Case, F, Max, OuterRef, Q, Subquery, When
 from django.db.models.query import QuerySet
 
 
 class MessageQuerySet(QuerySet):
-    def get_conversations_for_user(self, user: "User") -> "MessageQuerySet":
+    def get_conversations_for_user(self, user: "User") -> "MessageQuerySet":  # noqa
```

Comment:
Why is this `noqa`?

Reply:
Same as the other one: src/open_inwoner/accounts/query.py:6:48: F821 undefined name 'User'

Reply:
We can just import User from .account? The pluggable user model is not really a thing in the project.